lib/components/itemOption/itemOption.js

```diff
@@ -40,12 +48,14 @@ ItemOption.propTypes = {
   changeSwitch: PropTypes.func.isRequired,
   disabled: PropTypes.bool,
   hasValue: PropTypes.bool || PropTypes.string,
+  placementTooltip: PropTypes.string,
```

Comment:
Nitpick: Rename to `tooltipPlacement` for consistency with the other tooltip prop